app/controllers/OrganisationDetailsController.scala

```diff
@@ -85,12 +85,11 @@ class OrganisationDetailsController @Inject()(
               )
             },
             data => {
-              shortLivedCache.cache[OrganisationDetails](cacheId, OrganisationDetails.cacheKey, data).flatMap { _ =>
+              sessionCacheRepository.putSession[OrganisationDetails](DataKey(OrganisationDetails.cacheKey), data).flatMap { _ =>
                 logger.debug(s"BusinessStructure retrieved: ${businessStructure.businessStructure}")
                 rosmService.rosmRegister(businessStructure, data).flatMap {
-                  case Right(safeId) =>
-                    logger.debug("rosmRegister Successful")
```

Comment:
Did you mean to remove the `logger.debug` line? I think it's still useful to have

Reply:
ill add it back in 👍